src/common/crypto/mbedtls/cryptoprovider.cpp

```diff
@@ -641,6 +642,37 @@ Error MbedTLSCryptoProvider::ASN1DecodeOID(const Array<uint8_t>& inOID, Array<ui
     return crypto::ASN1RemoveTag(inOID, dst, MBEDTLS_ASN1_OID);
 }
 
+RetWithError<uuid::UUID> MbedTLSCryptoProvider::CreateSHA1(const uuid::UUID& space, const String& data)
+{
+    constexpr auto version = 5;
```

Comment:
shouldn't it be a static const in the classes's private section?

Reply:
I don't know, it is used for sha1 specific case only.

Reply:
renamed to cUUIDVersion